op-service/eth/ssz_test.go

```diff
@@ -330,6 +330,8 @@ func TestMarshalUnmarshalWithdrawals(t *testing.T) {
 			} else {
 				require.NoError(t, err)
 				require.Equal(t, input, output)
+				require.Equal(t, output.ParentHash, common.HexToHash("0x1234"))
+				require.Equal(t, output.ExtraData, BytesMax32(common.Hex2Bytes("0x6666")))
```

Comment:
The `require.Equal(t, input, output)` already checks the fields match. This test is focussed on ensuring withdrawals are preserved so has an explicit check for them, but we don't need these additional checks.